packages/ra-ui-materialui/src/button/CreateButton.js

```diff
@@ -69,15 +69,10 @@ const CreateButton = ({
 CreateButton.propTypes = {
     basePath: PropTypes.string,
     className: PropTypes.string,
-    classes: PropTypes.object,
     label: PropTypes.string,
     size: PropTypes.string,
     icon: PropTypes.element,
 };
 
-const enhance = compose(
-    onlyUpdateForKeys(['basePath', 'label', 'translate']),
-    withStyles(styles)
-);
-
+const enhance = onlyUpdateForKeys(['basePath', 'label', 'translate']);
```

Comment:
no need for an `enhance` variable if there is only one HOC: you can inline it with `export`

Reply:
Makes sense. Will make the changes.

Reply:
I've inlined the cases where there is only a single HOC